frontend/src/app/components/wp-form-group/wp-attribute-group.template.html

```diff
@@ -1,4 +1,4 @@
-<div class="-columns-2">
+<div class="can-have-columns -columns-2">
```

Comment:
In order to fulfill naming conventions this should probably be `-can-have-columns` as it is kind of a modifier class.

Reply:
Alright, thank you. I replaced the class name.